content-tree.d.ts

```diff
@@ -211,17 +211,27 @@ export declare namespace ContentTree {
         credit: string;
         picture?: ImageSetPicture;
     }
+    type TableColumnSettings = {
+        hideOnMobile: boolean;
+        sortable: boolean;
+        sortType: "text" | "number" | "date" | "currency" | "percent";
+    };
     interface Table extends Parent {
         type: "table";
-        children: [Caption | TableHead | TableBody];
-    }
-    interface Caption {
-        type: "caption";
-    }
-    interface TableHead {
-        type: "table-head";
-    }
-    interface TableBody {
-        type: "table-body";
+        title: string;
+        footer: string;
+        stripes: boolean;
+        compact: boolean;
+        layoutWidth: string;
+        rows: number;
+        columns: number;
+        collapseAfterHowManyRows: number;
+        responsiveStyle: "stacked" | "scroll";
```

Comment:
**Question:** Is this related to [**this**](https://github.com/Financial-Times/next-es-interface/blob/b77c1823ae8977b4a35902a160b3fbbe4c35b2b1/heroku/web/server/transformers/plugins/article/concerns/transform-data-tables.js#L189)? If so, should this actually be ↓?
```suggestion
        responsiveStyle: "auto" | "horizontal-scroll" | "stacked";
```